tests/testthat/test_synthdid.R

```diff
@@ -1,7 +1,9 @@
 test_that("a simple workflow doesn't error", {
```

Comment:
The last PR ran package tests in [4 minutes](https://dev.azure.com/grf-labs/synth-inference/_build/results?buildId=782&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9).

In this PR they take 35 minutes...

Reply:
Your "paper-results.Rmd" builds on Azure https://synth-inference.github.io/synthdid/articles/paper-results.html, but the total build time now is close to 1 hr, which is the maximum Azure allows